sql/src/test/java/io/crate/analyze/where/WhereClauseAnalyzerTest.java

```diff
@@ -23,86 +23,60 @@
-import io.crate.metadata.*;
+import io.crate.metadata.PartitionName;
+import io.crate.metadata.Routing;
+import io.crate.metadata.TableIdent;
+import io.crate.metadata.TransactionContext;
 import io.crate.metadata.doc.DocSchemaInfo;
-import io.crate.metadata.sys.MetaDataSysModule;
 import io.crate.metadata.table.ColumnPolicy;
-import io.crate.metadata.table.SchemaInfo;
-import io.crate.metadata.table.TableInfo;
 import io.crate.metadata.table.TestingTableInfo;
-import io.crate.operation.operator.OperatorModule;
 import io.crate.operation.operator.any.AnyEqOperator;
 import io.crate.operation.operator.any.AnyLikeOperator;
-import io.crate.operation.predicate.PredicateModule;
-import io.crate.operation.scalar.ScalarFunctionModule;
-import io.crate.sql.parser.SqlParser;
 import io.crate.test.integration.CrateUnitTest;
-import io.crate.testing.MockedClusterServiceModule;
+import io.crate.testing.SQLExecutor;
 import io.crate.types.ArrayType;
 import io.crate.types.DataType;
 import io.crate.types.DataTypes;
 import io.crate.types.SetType;
 import org.apache.lucene.util.BytesRef;
-import org.elasticsearch.common.inject.Injector;
-import org.elasticsearch.common.inject.ModulesBuilder;
-import org.elasticsearch.threadpool.ThreadPool;
+import org.elasticsearch.test.cluster.NoopClusterService;
 import org.hamcrest.Matchers;
-import org.junit.After;
 import org.junit.Before;
-import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
-import org.mockito.Mock;
 
-import java.util.*;
-import java.util.concurrent.TimeUnit;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
 
 import static io.crate.testing.TestingHelpers.*;
 import static org.hamcrest.Matchers.*;
-import static org.mockito.Mockito.when;
 
 @SuppressWarnings("unchecked")
 public class WhereClauseAnalyzerTest extends CrateUnitTest {
 
-    public static final String GENERATED_COL_TABLE_NAME = "generated_col";
-    public static final String DOUBLE_GEN_PARTITIONED_TABLE_NAME = "double_gen_parted";
-
-    @Rule
-    public ExpectedException expectedException = ExpectedException.none();
-
-    private Analyzer analyzer;
-    private ThreadPool threadPool;
+    private static final String GENERATED_COL_TABLE_NAME = "generated_col";
+    private static final String DOUBLE_GEN_PARTITIONED_TABLE_NAME = "double_gen_parted";
 
+    private final Routing twoNodeRouting = new Routing(TreeMapBuilder.<String, Map<String, List<Integer>>>newMapBuilder()
+        .put("nodeOne", TreeMapBuilder.<String, List<Integer>>newMapBuilder().put("t1", Arrays.asList(1, 2)).map())
+        .put("nodeTow", TreeMapBuilder.<String, List<Integer>>newMapBuilder().put("t1", Arrays.asList(3, 4)).map())
+        .map());
     private final TransactionContext transactionContext = new TransactionContext();
-
-    @Mock
-    private SchemaInfo schemaInfo;
-    private Functions functions;
+    private SQLExecutor e;
```

Comment:
Ok, then consistency is more important I just don't like short names like that for class attributes, for local variables it's ok and e is usually what we use for exception that's why I mentioned, but I leave up to you!